deployer/prepare.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
+# Patch crates to be on same versions
+mkdir -p $CARGO_HOME; \
+echo '[patch.crates-io]
```

Comment:
As a result of this, the patch is not applied to other images. I'm guessing this is fine because the only one that needs patched up deps is deployer?

Reply:
Yip, we've only actually needed it for deployer